src/BasicRum/Beacon/Catcher/Storage/File.php

```diff
@@ -62,6 +62,8 @@ public function generateBundleFromRawBeacons(): int
                 'id' => basename($filePath),
                 'beacon_data' => file_get_contents($filePath),
             ];
+
+            $this->deleteRawBeaconFile($filePath);
```

Comment:
Hello @korzol 

Could we call outside of this foreach? This function can accept $beaconFiles as an argument and delete all files listed in $beaconFiles.

Reply:
Hello @ceckoslab 
I made it the way you requested. But under some conditions it may break data consistency. I guess we have to discuss and work out some policy or the way we can guarantee the data stored in bundle files and DB is consistent